### PR TITLE
Nativeshortuuidfield/v1

### PR DESCRIPTION
**Pull request by laithfawwaz**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+Copyright (c) 2020 FounderTherapy
```
Comment (danaspiegel) [inline LICENSE]: https://help.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository

Use the MIT License

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+django-nativeshortuuidfield
+----------------
+
+Provides a NativeShortUUIDField for your Django models which uses the base-57 "Short UUID" package at https://github.com/stochastic-technologies/shortuuid/ to be used in Python
+and Store it as full UUID in database.
```
Comment (danaspiegel) [inline README.rst]: lower case "Store"

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+============
+
+Install it with pip (or easy_install)::
+
+	pip install django-nativeshortuuidfield
+
+Usage
+=====
+
+First you'll need to attach a NativeShortUUIDField to your class::
```
Comment (danaspiegel) [inline README.rst]: add, not "attach"

File: LICENSE (hunk 1 above)
Comment (laithfawwaz) [reply LICENSE]: Updated

File: README.rst (hunk 3 above)
Comment (laithfawwaz) [reply README.rst]: Updated

File: README.rst (hunk 2 above)
Comment (laithfawwaz) [reply README.rst]: Updated

**Review by danaspiegel — APPROVED** [review]

**Pull request merged**